addons/hr/hr.py

```diff
@@ -316,10 +316,11 @@ def onchange_department_id(self, cr, uid, ids, department_id, context=None):
         return {'value': value}
 
     def onchange_user(self, cr, uid, ids, user_id, context=None):
-        work_email = False
         if user_id:
-            work_email = self.pool.get('res.users').browse(cr, uid, user_id, context=context).email
-        return {'value': {'work_email': work_email}}
+            user_obj = self.pool.get('res.users').browse(cr, uid, user_id, context=context)
```

Comment:
Looks ok, but this is a new feature, so it should target `master`, not 8.0 (in order to comply with our [stable policy](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean)).
We can cherry-pick it in master in order to save you the extra PR, if you like.

Some (minor) extra nitpicking:
 - `user_obj` is a user browse_record, so it should be named `user`
 - since you're touching those lines already, you can simplify `self.pool.get('res.users')` into `self.pool['res.users']`
 - the dict returned with the data would be much better split into multiple indented lines
 - the dict returned when there is no `user_id` can simply be `{}`, the empty `value` is unnecessary

Thanks for the contribution!